src/components/Card/Card.js

```diff
@@ -9,7 +9,7 @@ export const Card = ({ image, brand, colorway, hue }) => {
       </section>
       <section className="polish-details">
         <h3 className="brand-name">Brand: {brand}</h3>
-        <h3 className="colorway">Colorway: {colorway}</h3>
+        {colorway && <h3 className="colorway">Colorway: {colorway}</h3>}
```

Comment:
Are the conditionals for colorway and image going to mess up your styling if there isn't an image or colorway? If so, maybe consider adding some defaults in your cleaning function for filler content like an "image not found" photo and "colorway not found" text

Reply:
They are already built in and accounted for in styling as of now - I decided to make these inputs optional on the user end after I built out the original card. I haven't come across any missing data from the api as of yet.